airflow/api_connexion/endpoints/dag_run_endpoint.py

```diff
@@ -18,7 +18,7 @@
 
 from http import HTTPStatus
 
-import pendulum
```

Comment:
The `pendulum` module must be loaded in either case here. I don't think you will see a performance gain from a change like this.

I ran a very quick couple of commands to test this:
![Screenshot from 2023-03-31 13-20-06](https://user-images.githubusercontent.com/65743084/229222012-9c22672a-d438-4b43-a544-0f38d381604f.png)

You can see that they're the same (even the `from` case being slower, but this is normal variance of course).

Reply:
Indeed, speed-wise seems there is no real difference. However, importing the whole pandas namespace to use only DataFrame seems excessive. What do you think @o-nikolas ?

Reply:
That’s not how Python works. `import pandas` and `from pandas import DataFrame` has exactly the same import time.